app/views/pages/index.html.haml

```diff
@@ -49,11 +49,11 @@
     %p
       WiCHacks is an all women 24-hour hackathon hosted by Women in Computing at RIT. A
       %b hackathon
-      is a collaborative programming event in which participants create an app, website, game, or other piece of software over the course of the event.  Hackathon participants can be female college (including non-RIT) or high school students who are at least 16 years of age.
+      is a collaborative programming event in which participants create an app, website, game, or other piece of software over the course of the event.  
     %p
       %b> This event is open to women of all skill levels
-      \: those who haven’t programmed at all, women who have done some programming, and the best women programmers out there. It will be a great day to learn, invent, and create the future.  Don’t have a team in mind?  No worries -- RIT Women in Computing will be hosting a series of events (including one on the morning of the event) to facilitate team formation.
-      %b All meals (breakfast, lunch, dinner, and snacks) will be provided, and participation is free.
+      \: those who haven’t programmed at all, women who have done some programming, and the best women programmers out there. It will be a great day to learn, invent, and create the future.  Don’t have a team in mind?  No worries -- RIT Women in Computing will be hosting a series of events (including one on the morning of the event) to facilitate team formation. All meals (breakfast, lunch, dinner, and snacks) will be provided, and participation is free!
+      %b "All participants must be female college students (at RIT or another university)."
```

Comment:
The beginning & ending quotes can be removed

Reply:
okie!